src/components/FeaturePanel/Climbing/ClimbingView.tsx

```diff
@@ -306,6 +308,20 @@ export const ClimbingView = ({ photo }: { photo?: string }) => {
     loadPhotoRelatedData();
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
+  React.useEffect(() => {
+    const downHandler = (e) => {
+      if (e.key === 'h') {
+        setIsRoutesLayerVisible(!isRoutesLayerVisible);
+      }
+    };
+
+    window.addEventListener('keydown', downHandler);
+
+    return () => {
+      window.removeEventListener('keydown', downHandler);
+    };
+  }, [isRoutesLayerVisible, setIsRoutesLayerVisible]);
+
```

Comment:
Could this be please extracted as a cusom hook? 🙂

Reply:
I see it is done here:
https://github.com/zbycz/osmapp/pull/1105/files#diff-c546452ec9d2c506cb5d5bbe315ba74725be259f03aee5f197665cee1b3e8adf